sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
-
+## 1.3.0-beta.1 (2020-09-28)
+- Updated dependency version of `azure-messaging-eventhubs` to `5.3.0-beta.1`.
```

Comment:
This might need sectioning according to new guideline.

Reply:
Also, do add this to the September release notes.

Reply:
@YijunXieMS - Spoke to @samvaity  about this and we'll not add any section for this as it's only a dependency update and doesn't fit into any section.